dsgrid/dimension/standard.py

```diff
@@ -72,6 +78,16 @@ class Subsector(SubsectorDimensionBaseModel):
 # ---------------------------
 # METRIC DIMENSIONS
 # ---------------------------
+    
+class FunctionalForm(str, Enum):
```

Comment:
Apparently, we have not updated all enums. Python 3.11 added StrEnum.
```suggestion
class FunctionalForm(StrEnum):
```